test/unit-test/test_lib/test_analysis/test_user.py

```diff
@@ -57,16 +57,52 @@ def test_nick_change_graph(self, mock_get_year_month_day, mock_build_graphs, moc
     @patch("lib.config.DEBUGGER", new = True)
     @patch("lib.config.PRINT_WORDS", new = True)
     def test_top_keywords_for_nick(self):
-        user_keyword_freq_dict = util.load_from_disk(current_directory + "/data/user_test/user_keyword_freq_dict")
-        expected_top_keywords = util.load_from_disk(current_directory + "/data/user_test/top_keywords")
```

Comment:
top_keywords and top_keywords_normalized files are not used anymore in this test. If they remain unused in other tests as well, we should remove them.